packages/common/src/authentication/AuthContext.tsx

```diff
@@ -76,12 +76,11 @@ const useRefreshingAuth = (
 ) => {
   const { setHeader } = useGql();
   setHeader('Authorization', `Bearer ${token}`);
-  const { data, isSuccess } = useGetRefreshToken(token ?? '');
+  const { data, enabled, isSuccess } = useGetRefreshToken(token ?? '');
```

Comment:
this isn't great. The callback is setting state on the component, causing a re-render which will fire the `useEffect` again. fine, except that the `useGetRefreshToken` is only returning data after a timeout ( to prevent issues when developing.. without that the refresh will wipe out your token and log you out ). until the refresh returns, the `data` is undefined and isn't useful as a referential dependency.

all of which is to say.. this is working but I'm not that happy with it. 🤷